samples/kvs_gstreamer_sample.cpp

```diff
@@ -694,7 +696,8 @@ int gstreamer_live_source_init(int argc, char* argv[], CustomData *data, GstElem
     // Attempt to create vtenc encoder
     encoder = gst_element_factory_make("vtenc_h264_hw", "encoder");
     if (encoder) {
-        source = gst_element_factory_make("autovideosrc", "source");
+        LOG_DEBUG("Using videotestsrc")
```

Comment:
This log is misleading so it's only using `videotestsrc` if we were able to successfully create source and it's not null..  we're just trying `vidoetestsrc` here.  Further I would move up setting vtenc = true to the first line inside the if(encoder) { section since if encoder is TRUE we know we've successfully created it.  Shouldn't really change logic much but I think the logic flow is better reading the code otherwise it kind of reads like vtenc is being set as a result  of creating videotestsrc.

Reply:
Sure thing. Done.